users/migrations/0002_static_data.py

```diff
@@ -1,22 +1,26 @@
 from datetime import datetime
 
 from django.db import migrations, transaction
+from django.contrib.auth.hashers import make_password
 
 
 def create_initial_users(apps, schema_editor):
     Profile = apps.get_model('users', 'Profile')
     User = apps.get_model('auth', 'User')
     profile_data = [
-        ('Ofek', 'qaz[;.123', datetime(2020, 5, 17), '050-123456789'),
-        ('Danny', 'qaz[;.123', datetime(2020, 5, 17), '050-223456789'),
-        ('Rimon', 'qaz[;.123', datetime(2020, 5, 17), '050-323456789'),
-        ('Rotem', 'qaz[;.123', datetime(2020, 5, 17), '050-423456789'),
-        ('Saar', 'qaz[;.123', datetime(2020, 5, 17), '050-523456789'),
-        ('Tal', 'qaz[;.123', datetime(2020, 5, 17), '050-623456789'),
+        ('Ofek', 'ofekPassword', datetime(2020, 5, 17), '050-123456789', 'Manor'),
+        ('Danny', 'dannyPassword', datetime(2020, 5, 17), '050-223456789', 'Senderovych'),
+        ('Rimon', 'rimonPassword', datetime(2020, 5, 17), '050-323456789', 'Shushan'),
+        ('Rotem', 'rotemPassword', datetime(2020, 5, 17), '050-423456789', 'Eidlitz'),
+        ('Saar', 'saarPassword', datetime(2020, 5, 17), '050-523456789', 'Orus'),
+        ('Tal', 'talPassword', datetime(2020, 5, 17), '050-623456789', 'Koren'),
     ]
     with transaction.atomic():
-        for username, password, dob, phone_number in profile_data:
-            user = User(username=username, password=password)
+        for username, password, dob, phone_number, last_name in profile_data:
+            encrypted_pw = make_password(password)
+            user = User(username=username, password=encrypted_pw)
+            user.first_name = username
+            user.last_name = last_name
```

Comment:
Please set the first and last name as part of the user creation like
```suggestion
            user = User(username=username, password=encrypted_pw, first_name=username, last_name=last_name)
```